chat-commands.js

```diff
@@ -2274,9 +2274,11 @@ exports.commands = {
 			target = targets[1].trim();
 			roomId = toId(targets[0]) || room.id;
 		}
+		let targetRoom = Rooms.get(roomId);
+		let addModlogLinks = Config.modloglink && (!hideIps || (Config.roomauthModlogLink && targetRoom && !targetRoom.isPrivate));
```

Comment:
Config.roomauthModlogLink is a boolean setting. If this is used by other servers besides PS main, they might not want to give roomauth access.